rust/azure_iot_operations_connector/src/deployment_artifacts/connector.rs

```diff
@@ -459,12 +528,12 @@ mod tests {
     use test_case::{test_case, test_matrix};
 
     /// Simulates a file mount directory using a temporary directory.
-    struct TempMount {
+    pub struct TempMount {
```

Comment:
pub changes still needed here now that the files were merged back together?

Reply:
ah, good catch let me remove